src/test/scala/io/opentargets/etl/backend/DrugBeta/IndicationTest.scala

```diff
@@ -85,14 +86,14 @@ class IndicationTest
 
   "Processing ChEMBL indications data" should "correctly process raw ChEMBL data into expected format" in {
     // given
-    val indicationDf = IndicationTest.indicationDf(sparkSession)
-    val efoDf = IndicationTest.efoDf(sparkSession)
+    val indicationDf: DataFrame = IndicationTest.indicationDf(sparkSession)
+    val efoDf: DataFrame = IndicationTest.efoDf(sparkSession)
 
-    val indicationInst = new Indication(indicationDf, efoDf)(sparkSession)
+    val indication: Indication = new Indication(indicationDf, efoDf)(sparkSession)
```

Comment:
@JarrodBaker this case is quite straight and there is no need for using `new` as you are using a class just to hold a method with no state. I would suggest just simplify it putting the functions inside the singleton object. Also, you can use the function `spark.Helpers.stripIDFromURI` here in another [PR](https://github.com/opentargets/platform-etl-backend/pull/55/files#diff-721c0005f76b04f52eaf3eb3dfdfa4e58fd0f5f8a9b3bb37c697a48efca58dd2R69) and remove the one you are created for this. Mine is using the native spark column functions

If you want to encapsulate the function and use it later you can return a function with all configure and then curry it with `_` at the right outside the function